Changelog.md

```diff
@@ -1,3 +1,9 @@
+## 1.7.3 - unreleased
+
+### Changed
+
+-   Update dependencies to support nRF Connect for Desktop v5.2.0 release.
```

Comment:
v5.2.0 - future launcher release version
I do not state that we support nRF Connect for Desktop version 5.2.0, as app will be compatible with previous releases as well.